shared/js/default_activity_helper.js

```diff
@@ -0,0 +1,151 @@
+/* global SettingsHelper */
+/* exported DefaultActivityHelper */
+
+(function(exports) {
+  'use strict';
+
+  var supportedActivities = [
+    {
+      name: 'pick',
+      type: ['image/jpeg'],
```

Comment:
ooops, didn't pay attention. I don't know about any restriction so far, so I'll change filters on image / video / audio to `<type>/*`